packages/xo-web/src/common/intl/messages.js

```diff
@@ -592,6 +592,10 @@ const messages = {
   remotePlaceHolderPassword: 'Password(fill to edit)',
 
   // ------ New Storage -----
+
+  newSr: 'New SR',
+  newSrConfirm:
+    'This will erase the entire disk ({name}) or partition to create a new storage repository. Are you sure you want to continue?',
```

Comment:
```suggestion
    'This will erase the entire disk or partition ({name}) to create a new storage repository. Are you sure you want to continue?',
```

Name can also correspond to a partition, therefore it should be after `disk or partition`.